torch/csrc/inductor/aot_runtime/model.h

```diff
@@ -6,27 +6,40 @@
 #include <unordered_map>
```

Comment:
I need a reminder if this file is ok to use in model.so or is predictor only LOL

Reply:
It used to have ATen includes but it looks like... you got rid of all of them? So it's model.so now?

Reply:
Yes, it is for model.so only. I will add warnings in those model.so only headers to not include aten/c10 headers.